core/src/main/java/org/commonjava/maven/ext/manip/state/RESTState.java

```diff
@@ -33,11 +33,8 @@
     public RESTState( final Properties userProps )
     {
         restURL = userProps.getProperty( "restURL" );
-        /*
-      Limit on maximum count to make in the REST call.
-     */
-        int maxRestSize = Integer.parseInt( userProps.getProperty( "restMaxSize", "0" ) );
```

Comment:
If we're removing restMaxSize the documentation in gh-pages will need updating (normally a separate PR).